guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -24,6 +24,13 @@ ifdef::satellite[]
 * {Project} {ProjectVersion} running on {RHEL} 7.
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* If you run {Project} in a disconnected environment, you must obtain and deploy Leapp metadata manually. For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+* You require access to {RHEL} and {Project} packages.
+* Follow the steps for obtaining ISO files for {RHEL} 8 and {Project} in https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html-single/upgrading_and_updating_red_hat_satellite/index#upgrading_a_disconnected_satellite[Upgrading a Disconnected {Project}].
```

Comment:
```suggestion
* Follow the steps for obtaining ISO files for {RHEL} 8 and {Project} in https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersion}/html-single/upgrading_and_updating_red_hat_satellite/index#upgrading_a_disconnected_satellite[Upgrading a Disconnected {Project}].
```

Reply:
The more I think about it the more I think we need to lay this out explicitly in this guide. Our disconnected guide is all about how to acquire ISO for the next version of Satellite and set it up on disk. In this workflow, we need users to:

 1) Acquire RHEL 8 ISO
 2) Acquire Satellite 6.11 RHEL 8 ISO
 3) Unpack them to location and repository files that are parallel to but do not over write the existing repository setup
 4) Run leapp commands with the specified RHEL 8 and Satlelite on RHEL 8 repository IDs

If we do not configure the system with the RHEL 8 repositories in parallel then we run the risk of a user running `leapp preupgrade`, running into problems they need to address but no longer having access to the RHEL 7 repositories and Satellite RHEL 7 repositories that may be needed while they address their identified `leapp` issues. 

If it would help for me to break what I mean by this out into more detail, I am happy to.

Reply:
@ehelms I pushed another change but maybe we can clarify Step 3 discussing unpacking the ISO files. This latest change I made doesn't specifically call that out.

Reply:
In the disconnected installation and upgrade guides we lay out the steps to unpack the ISOs in a predictable manner for the user, it feels like we should do the same here and reduce the amount of guess work on the user's part.

Reply:
@ehelms Any chance you might be able to break this out into more detail? I was going to pull the steps for unpacking ISOs from the Disconnected Satellite Server section in the Upgrading Guide but I want to make sure I use the correct content.